src/utils/audio_capture.py

```
@@ -43,24 +62,48 @@ def _open_handle(self, call_id: str, stream_name: str, sample_rate: int) -> wave
+            try:
+                if handle is None:
+                    wf = self._open_handle(call_id, stream_name, sample_rate)
+                    self._handles[key] = (wf, sample_rate)
+                else:
+                    wf, existing_rate = handle
+                    if existing_rate != sample_rate:
+                        # Close and reopen with new rate to avoid inconsistent headers
+                        try:
+                            wf.close()
+                        except Exception:
+                            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)